### PR TITLE
fix: remove components dir from sweep command

### PR DESCRIPTION
**Pull request by chrisfalaska**
# Alaska Airlines Pull Request

The `sweep` command was updated in [341](https://github.com/AlaskaAirlines/auro-input/pull/341) to search for the `/components/` folder, which is currently exclusive to the new [auro-form](https://github.com/AlaskaAirlines/auro-form/) component under development as of https://github.com/AlaskaAirlines/auro-form/issues/1.

**Resolves:** 356

## Summary:

- Removed the `/components/` folder from the `sweep` command
- Returned the `rm` command to remove the `dist` and `demo/css` folders

## Type of change:

Please delete options that are not relevant.

- [X] Revision of an existing capability

## Checklist:

- [X] My update follows the CONTRIBUTING guidelines of this project
- [X] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

## Summary by Sourcery

Bug Fixes:
- Remove the 'components' directory from the 'sweep' command in the package.json to prevent unintended file deletions.

## Summary by Sourcery

Bug Fixes:
- Remove the 'components' directory from the 'sweep' command in the package.json to prevent unintended file deletions.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -176,7 +176,7 @@
     "postinstall": "node packageScripts/postinstall.mjs",
     "sass:render": "sass-render 'src/**/*.css' 'components/**/*.css' -t ./node_modules/@aurodesignsystem/auro-library/scripts/build/staticStyles-template.js",
     "serve": "web-dev-server --open demo/ --node-resolve --watch",
-    "sweep": "find ./demo ./dist ./src ./components -type f \\( -name \"*.css\" -o -name \"*-css.js\" \\) -delete",
+    "sweep": "rm -rf ./demo/css ./dist | find ./demo ./dist ./src ./-type f \\( -name \"*.css\" -o -name \"*-css.js\" \\) -delete",
```
Comment (jason-capsule42) [inline package.json:179]: This isn't really a folder is it? `./demo/css`

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (chrisfalaska) [reply package.json:179]: I wasn't sure. It was in the original file, so I restored it:

https://github.com/AlaskaAirlines/auro-input/pull/341/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L179

Should I remove it?

**Comment by chrisfalaska** [comment]
@sourcery-ai review

File: package.json (hunk 1 above)
Comment (chrisfalaska) [reply package.json]: Removed, thanks

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**